server/src/main/java/org/elasticsearch/search/aggregations/bucket/composite/GeoTileGridValuesSourceBuilder.java

```diff
@@ -61,13 +70,27 @@ static GeoTileGridValuesSourceBuilder parse(String name, XContentParser parser)
     GeoTileGridValuesSourceBuilder(StreamInput in) throws IOException {
         super(in);
         this.precision = in.readInt();
+        if (in.getVersion().onOrAfter(Version.V_8_0_0)) {
```

Comment:
should we backport to 7.x?

Reply:
for sure. I tend to go this strategy when doing bwc changes

https://github.com/elastic/elasticsearch-team/pull/318/files#diff-60ab984f14487976eab0e31f39880a3bR30

Reply:
but this does remind me, I do not have enough tests for this bwc change. so I will get on that